Makefile

```
@@ -287,7 +287,7 @@ lint: $(BUILD_DIRS)
 	    --env GO111MODULE=on                                    \
 	    --env GOFLAGS="-mod=vendor"                             \
 	    $(BUILD_IMAGE)                                          \
-	    golangci-lint run --enable $(ADDTL_LINTERS) --skip-dirs-use-default --deadline=10m
+	    golangci-lint run --enable $(ADDTL_LINTERS) --skip-dirs-use-default --deadline=10m --skip-dirs=client,vendor --skip-files="(generated.*\.go$\)"
```

we can remove braces now, as there's only one rule
```suggestion
	    golangci-lint run --enable $(ADDTL_LINTERS) --skip-dirs-use-default --deadline=10m --skip-dirs=client,vendor --skip-files="generated.*\.go$\"
```